test/acceptance/public-api-test.js

```diff
@@ -1118,15 +1131,16 @@ describe('public api', function () {
       project.dispose();
     });
 
-    it('[.html] does not identify errors for ember-cli default app/index.html (3.20)', async function () {
+    it('[.html] does not identify errors (except for no-forbidden-elements) for ember-cli default app/index.html (3.20)', async function () {
       // reset config to default value
       project.setConfig();
```

Comment:
Calling `project.setConfig();` here previously had no effect because the config was already loaded when the `Linter` was constructed in `beforeEach`, so this test wasn't actually using the full recommended config. Now this does have an effect since the config is loaded when `verify` is called later in this test. There are actually violations of `no-forbidden-elements` and `require-lang-attribute` so I had to disable or fix those.